Assets/Project/Scripts/Modules/MenuSelectScene/LevelSelect/SavableScrollRect.cs

```diff
@@ -31,13 +33,16 @@ protected override void Awake()
             _RIGHT_OFFSET = content.anchorMax.x - content.pivot.x;
             _TOP_OFFSET = content.anchorMax.y - content.pivot.y;
             _BOTTOM_OFFSET = Mathf.Abs(content.anchorMin.y - content.pivot.y);
+
+            // 明示的にUnityEvent使用することを宣言
+            _transformGesture.UseUnityEvents = true;
+            _transformGesture.OnTransformStart.AsObservable().Subscribe(OnTransformStarted).AddTo(this);
```

Comment:
```suggestion
            _transformGesture.OnTransformStart.AsObservable()
                .Subscribe(_ => {
                    // 2点タッチしている時はスクロールしない
                    horizontal = false;
                    vertical = false;
                })
                .AddTo(this);
```
にして `private void OnTransformStarted(Gesture gesture)` を消したい

Reply:
e4b9568